### PR TITLE
codeowners generator

### LINKED ISSUE #5: Add codeowners generator to composer-monolith command

### PR DESCRIPTION
**Pull request by nusje2000**
fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,14 @@ To update a dependency you can use
 This will make sure the version constraints in all sub- and rootpackages
 match the provided version_constraint. This will not actually update the
 package using composer, this must still be done manually.
+
+### CODEOWNERS file
+In large monolithic repositories, the maintainability can be dificult. Using a CODEOWNERS file can let
```
Comment (nvdbeek) [inline README.md]: difficult

File: src/Validator/Validator.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ public function __construct(RuleCollection $rules)
         $this->rules = $rules;
     }
 
-    public function validate(DependencyGraph $graph): ViolationCollection
+    public function getErrors(DependencyGraph $graph): ViolationCollection
```
Comment (nvdbeek) [inline src/Validator/Validator.php]: The method name `getErrors` is not exactly what you are doing. `Errors != Violations`

File: README.md (hunk 1 above)
Comment (nusje2000) [reply README.md]: spelling is dificult okay :laughing:

File: src/Validator/Validator.php (hunk 2 above)
Comment (nusje2000) [reply src/Validator/Validator.php]: True, ill change it.

**Pull request merged**